src/main/java/com/softserveinc/dokazovi/scheduler/VerificationTokenCleanupScheduler.java

```diff
@@ -0,0 +1,17 @@
+package com.softserveinc.dokazovi.scheduler;
+
+import com.softserveinc.dokazovi.service.VerificationTokenService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class VerificationTokenCleanupScheduler {
+    private final VerificationTokenService verificationTokenService;
+
+    @Scheduled(cron = "0 0 0 * * ?")
```

Comment:
Unsure about this, but won't block review due to it.

Just as a note - this won't work if you start scaling the app horizontally, you need an external scheduler or to enable clustering mode with a spring-supported embedded scheduler